apps/web/src/server/views/inspector/issue-decision-success.njk

```diff
@@ -0,0 +1,30 @@
+{% extends "app/layouts/app-two-column.layout.njk" %}
+
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+{% set pageTitle = 'Decision issued' %}
+
+{% block pageHeading %}{% endblock %}
+{% block pageContent %}
+	{{
```

Comment:
Indentation looks a bit wonky here but it could just be github's rendering

Reply:
To be fixed with introduction of programmatic formatter.